### ISSUE TITLE
array - maxItems and minItems not displayed in generated html

### ISSUE THREAD

**Issue by muthulingapandian** [raml2html/raml2html#383]
Hi,

I defined below type in my RAML, but generated HTML doesn't display minItems & maxItems specified. Please suggest if I am doing anything wrong. or is it open bug, suppose to fix in upcoming release.

attribute:
	type: array
	items:
		type: string
	minItems: 1
	maxItems: 200

(Or) 

attribute:
	type: string[]
	minItems: 1
	maxItems: 200

Thank you

Regards
Muthu

**Comment by kevinrenskers** [comment]
Which version of raml2html did you use?

**Comment by muthulingapandian** [comment]
Hi Kevin,

I used 7.0.0 latest version.

**Comment by kevinrenskers** [comment]
Thanks, we'll look into this. Can you perhaps make the smallest possible RAML file that demonstrates the problem and share it here?

**Comment by muthulingapandian** [comment]
[sampleRaml.zip](https://github.com/raml2html/raml2html/files/1741168/sampleRaml.zip)
Please find attached 'api.raml' & generated 'api.html' using raml2html version 7.0.0

**Comment by kevinrenskers** [comment]
I have confirmed that raml2obj does correctly parse this, and so it's definitely part of the RAML object that's passed into the template. It's just that the template doesn't render these properties.

I've moved your issue to the default-theme repo, see [here](https://github.com/raml2html/default-theme/issues/18). It shouldn't be too hard to add this, so if you feel up to send a PR with this change, that would be awesome! :)
